spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -58,7 +99,7 @@ The Maven coordinates for this Eclipse release are as follows:
 <dependency>
     <groupId>org.eclipse.microprofile</groupId>
     <artifactId>microprofile</artifactId>
-    <version>3.2</version>
+    <version>3.3</version>
```

Comment:
Why did you change the version for the MP 3.2 description?  That doesn't look right...

Reply:
Fixed.